handlers/health.go

```diff
@@ -169,36 +177,36 @@ func (p *appHealthParams) extract(r *http.Request) {
 	p.App = vars["app"]
 }
 
-// workloadHealthParams holds the path and query parameters for WorkloadHealth
+// serviceHealthParams holds the path and query parameters for ServiceHealth
 //
-// swagger:parameters workloadHealth
-type workloadHealthParams struct {
+// swagger:parameters appHealth
```

Comment:
not sure what happened here... it seems you've switched positions of workloadHealthParams and serviceHealthParams, however why is the `swagger:parameters` `appHealth` and not `serviceHealth` here?

Reply:
Sorry again, yes. Will fix this sloppy c+p error.  I must have been in a daze.